File: presentation/src/main/java/com/everis/authspike/presenter/ContactsPresenter.java
package com.everis.authspike.presenter;

import com.everis.domain.model.LocalContact;

import java.util.List;

/**
 * Created by everis on 3/05/18.
 */

public interface ContactsPresenter extends BasePresenter {

    void getLocalContactBatch();
    void syncUserContactsByQuery(List<LocalContact> contacts);

    void syncUserContactsByRef(List<LocalContact> contacts);
}

File: presentation/src/main/java/com/everis/authspike/presenter/ContactsPresenter.kt
package com.everis.authspike.presenter

import com.everis.domain.model.LocalContact

interface ContactsPresenter : BasePresenter {

    fun getLocalContactBatch()
    fun syncUserContactsByQuery(contacts: List<LocalContact>)
    fun syncUserContactsByRef(contacts: List<LocalContact>)
    fun getContactsListConfig()
}
